src/components/FormProgressComponent.tsx

```diff
@@ -32,6 +32,7 @@ const FormProgressComponent: React.FC<FormProgressComponentProps> = ({
     >
       {steps.map((step, index) => (
         <FormProgress.Step
+          href="#"
```

Comment:
Dette kan muligens før til UU brudd. Kan du for sikkerhetskyld verifisere at det ikke dukker opp noe href="#" noe sted pga. den?

Reply:
Hadde fått beskjed fra Bjørn og Cecilie at dette var måten å gjøre det på, og de har godkjent det.

Reply:
Det svarte ikke helt på mitt spørsmål, og uten kontekst er det vanskelig for meg å vurdere om dette er lurt. Ved å kjøre applikasjonen fant jeg ikke hvordan denne property påvirker visningen. Hvis det er avtalt med Bjørn kanskje er det best at du tar review med ham?

Reply:
Siden det nå er en <a> uten href så vil den ikke havne inn i tab-rekkefølgen og man kan ikke nå stegvisningen med tastatur.